sawyer: #namespace
    n_actions: 14 # Increase and decrease 7 joints.

    work_space: # 3D cube in which Sawyers TCP ( right_electric_gripper_base frame) is allowed to move in
      x_max: 1.1
      x_min: 0.0
      y_max: 1.0
      y_min: -1.0
      z_max: 1.3
      z_min: 0.3

    max_effort: 50.0 # Maximum Effort , above which the episode is done.
    number_decimals_precision_obs: 1
    acceptable_distance_to_cube: 0.16 # Distance to the clock that we consider that it reached the block.
    tcp_z_position_min: 0.83 # Z value minimum to consider that the tcp is above the table, to avoid cheating the block distance done.
    done_reward: 1000.0 # reward
    closer_to_block_reward: 100.0 # reward